src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int firstLeg;
+    private int secondLeg;
+    private int thirdLeg;
+    private int fourthLeg;

```

Comment:
2 sides are enough to describe rectangle

Reply:
++